packages/cross_file/README.md

```diff
@@ -31,4 +31,4 @@ You will find links to the API docs on the [pub page](https://pub.dev/packages/c
 For help getting started with Flutter, view our online
 [documentation](http://flutter.io/).
 
-For help on editing plugin code, view the [documentation](https://flutter.io/platform-plugins/#edit-code).
\ No newline at end of file
+For help on editing plugin code, view the [documentation](https://flutter.io/platform-plugins/#edit-code).
```

Comment:
As long as you are touching this line, everything from line 29 on is boilerplate that should just be removed.

Reply:
@stuartmorgan, I didn't make any changes to line 34. I'm not really sure what change is git referring to. Do you want me to remove all the boilerplate from line 29 onwards? (I didn't completely understand your comment)

Reply:
> @stuartmorgan, I didn't make any changes to line 34. I'm not really sure what change is git referring to.

Looks like your editor made a whitespace change.

> Do you want me to remove all the boilerplate from line 29 onwards?

Yes.